CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/hierarchical/HierarchicalState.java

```diff
@@ -22,19 +22,19 @@
  */
 public final class HierarchicalState {
 
-    int steps;
-    int currentStep;
-    int optimumStep = 1;
-    FastNewman.Group[] groups;
-    boolean excludeSingleVertices;
-    boolean excludedElementsDimmed;
-    int[] clusterNumbers;
-    int[] clusterSeenBefore;
-    int redrawCount = 0;
-    transient long modificationCounter = -1;
-    transient long strucModificationCount = -1;
-    boolean interactive = true;
-    boolean colored = true;
+    public int steps;
+    public int currentStep;
+    public int optimumStep = 1;
+    public FastNewman.Group[] groups;
+    public boolean excludeSingleVertices;
+    public boolean excludedElementsDimmed;
+    public int[] clusterNumbers;
+    public int[] clusterSeenBefore;
+    public int redrawCount = 0;
+    public transient long modificationCounter = -1;
+    public transient long strucModificationCount = -1;
+    public boolean interactive = true;
+    public boolean colored = true;
```

Comment:
Is there a reason why these need to be made public? I would make these private and increase the accessibility as needed

Reply:
Just an oversight on my end, thinking if private would not work without getters and setters, then I would just use public. Makes sense to have them protected. 👍